CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## v0.8.0 UNRELEASED
+
+- **BREAKING:** `theme.config.useRootStyles` now defaults to `true` following
+  deprecation of `useBodyStyles`. Styles from `theme.styles.root` are now added
+  to `<html>` element by default.
+
+- Moved theme configuration options to `config` object in theme.
+
+  - **Deprecations:** `useRootStyles`, `useCustomProperties`,
+    `useColorSchemeMediaQuery`, `useBorderBox`, and `useLocalStorage` options on
+    the theme object are now deprecated, and they will be removed in a future
```

Comment:
```suggestion
    the theme object are now scoped under a `config` object on the theme, and the root-level options, now deprecated, will be removed in a future
```
Sounds like we've gotten rid of those options until you get a little further—don't want to concern anyone

Reply:
But this is exactly what deprecated means, doesn't it? "It still works, it's just not in fashion anymore. Go use the new stuff." :P (Yes, your version is way better. Thanks, mate :D)